src/molecules/Amount/Amount.native.js

```diff
@@ -96,24 +108,46 @@ const Amount = ({ size, testID, children, currency }) => {
   const [currencySymbol, amount] = formattedAmount.split(/\s/);
   const [integerPart, fractionPart] = amount.split('.');
 
+  const IntegerComponent =
+    size === 'xlarge' || size === 'xxlarge' || size === 'xxxlarge' ? Heading : Text;
```

Comment:
```suggestion
  const Component =
    (size === 'xlarge' || size === 'xxlarge' || size === 'xxxlarge') ? Heading : Text;
```
Integer is not proper name because for normal it has the whole amount `$123.00` while in camel it's just the `123`
Couldn't come up with a better name(Even AmountComponent doesn't fit). Open for anything else

Reply:
```PrimaryText```&  ```SecondaryText```